mig/install/generateconfs.py

```diff
@@ -345,7 +345,7 @@ def main(argv, _generate_confs=generate_confs):
         else:
             print('Error: command line option %r not supported!' % opt_name)
             usage(names)
-            sys.exit(1)
```

Comment:
Ugh, was this one case that crept through in the previous changes here?

Reply:
yes, I was quite certain that this was intended changed to return as well. Even if perhaps improbable to happen in unit tests where it matters.